src/user-guide/interventions/index.rst

```diff
@@ -523,3 +523,38 @@ I want to Skip a cycle of tasks and allow the workflow to continue
    ``-n root`` matches all tasks in a cycle. Similarly, it is possible to
    broadcast this setting to the root namespace of any family, or to
    multiple named tasks.
+
+
+Remove Tasks
+------------
+
+:Example:
+   I triggered tasks I did not mean to. They may have run-on. I want to undo
+   this.
+
+:Solution:
+   Use ``cylc remove`` to remove unwanted tasks.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: remove.gif
+         :width: 75%
+
+      .. warning::
+
+         Internal implemenation details mean that the task icon will not be
```

Comment:
Anyone think of a better way to describe this?

Reply:
I definitely wouldn't say it as you've written it - it makes it sound as if the GUI has unfixed bug or just can't represent the reality of the situation. In addition to noting the greying out, I think we should say something like this to indicate that the potential visibility of "removed" tasks is deliberate and correct:

```
Removing a task might not make it disappear from the GUI, because the
GUI displays all tasks in a graph-based window around current active tasks.
```

Reply:
(Note Ronnie's grey-out (flow-nums) branch is now merged)

Reply:
Yes, I suggest changing to a note, and I endorse ~Hilary for president~ Hilary's suggestion below. The GIF ought to be re-recorded